src/runtime/asm_arm64.s

```diff
@@ -15,17 +15,28 @@ TEXT runtime·rt0_go(SB),NOSPLIT,$0
 	MOVW	R0, 8(RSP) // argc
 	MOVD	R1, 16(RSP) // argv
 
-	// create istack out of the given (operating system) stack.
-	// _cgo_init may update stackguard.
+	
```

Comment:
Is there an easy way to regression test this? i.e. make sure this doesn't break ARM64 Linux.

Reply:
I don't believe it should be a problem but it's on my list. I will look into it at some point.